UnityAds/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 ## Changelog
+  * 3.7.5.1
+    * Add support for header bidding
```

Comment:
I will change to a more discrete statement
@andrewkonecny  @sean-coleman613 let me know what to fill in here

Reply:
Maybe something like:
`Fixed issue affecting creative rendering for DSP creatives (edge case)` 
... or is that too cagey? @andrewkonecny

Reply:
```suggestion
    * Update to Latest Unity APIs
```

Should encompass Load/Show changes as well as indirectly including header bidding. Most things are fine here as long as they aren't directly mentioning header bidding 😄